distribution/changelog.txt

```diff
@@ -12,6 +12,7 @@
 - Feature: [#13583] [Plugin] Add allowed_hosts to plugin section of config.
 - Feature: [#13614] Add terrain surfaces from RollerCoaster Tycoon 1.
 - Fix: [#12895] Mechanics are called to repair rides that have already been fixed.
+- Fix: [#13102] Underflow on height chart (Ride measurements).
```

Comment:
Isn't this just a regression?

Reply:
#13644 is, but #13102 has been in the game since the 0.3.1 release.